### DISCUSSION TITLE
DID Export

### DISCUSSION THREAD

**Discussion by ed-curran** [decentralized-identity/veramo#1294] [Q&A]
Hi, is there way to export a did managed by veramo? The notes [here](https://veramo.io/blog/version_3.0_release/#whats-new-about-key-management) and [here](https://github.com/decentralized-identity/veramo/discussions/589) suggest there's no functionality for this as of 3.0 but couldn't find anything more recent.

I was hoping to have DIDs managed primarily with veramo then imported into web5.js so that I can build some things with Decentralized Web Nodes.

**Comment by mirceanis** [comment]
Ideally there should be a way to use the same key manager for both so that you wouldn't have to move keys around, or even better, a veramo plugin for dwn.

In the meantime an acceptable solution might be to create the keys outside of veramo and import them.
